db/schema.rb

```diff
@@ -17,8 +17,8 @@
     t.integer  "user_id"
     t.datetime "created_at",                   null: false
     t.datetime "updated_at",                   null: false
-    t.text     "description"
     t.integer  "workspaces_count", default: 0
+    t.text     "description"
```

Comment:
Not sure. I didn't directly edit this file, though I did pull from master and run the migrations.